plugins/FeatureFlags/Features/Example.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/**
+ * Matomo - free/libre analytics platform
+ *
+ * @link    https://matomo.org
+ * @license https://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+
+namespace Piwik\Plugins\FeatureFlags\Features;
+
+use Piwik\Plugins\FeatureFlags\Feature;
+
+class Example extends Feature
```

Comment:
Just a comment here — I usually hate when the class name itself doesn't convey what it is about and one needs to look at the FQCN to understand the context from the namespace, but I get it here we don't want to have every feature named SomethingFeature as it may be e.g. RedMenuBar.

Question - Should we rename the classes representing the feature flags to SomethingFeatureFlag? Then the flag is disabled or enabled. But I'd be ok with having it either way.

Reply:
I can see that being annoying, I've updated to the suggestion you made 👍

Reply:
I understand your point @michalkleiner. But this might be a more general discussion and decision we might require here. Almost all classes in Matomo need the namespace for context. Like every plugin has a class called API, or most Diagnostic classes across the plugins don't have a name that end on Diagnostic. So starting that here might be somehow inconsistent, unless we decide to change that everywhere else as well over time.

In my opinion a class shouldn't duplicate it's namespace name. What would the namespace be good for then, and why not placing all classes in the root namespace & directory? 
If a class is used somewhere where the context might be unexpected it's imho more wise to use an alias in the use statement to make it easier to understand.

Reply:
Yep, I had a similar discussion with Dizzy some time last year. I think it's worth reopening at some point whether we want to change it or not, and if not, how we make sure we are at least consistent.
I agree that the class name should not replicate the full namespace, but not even knowing if it's a flag or controller or api response handler, broadly, is not helpful either. I'm happy to leave it here as is and we can discuss this at some point in future.